src/components/dashboard/Dashboard.js

```diff
@@ -160,34 +149,55 @@ const Dashboard = props => {
   const { avatar, fullName } = gdstore.get('profile')
   const feeds = gdstore.get('feeds')
 
-  log.info('LOGGER FEEDS', { feeds })
+  // TODO: Calculate scroll position to update Dashboard avatar, name and gd amount view
+  const scrollPos = 100
 
+  log.info('LOGGER FEEDS', { feeds })
   return (
     <View style={styles.dashboardView}>
       <TabsView goTo={navigation.navigate} routes={screenProps.routes} />
       <Wrapper backgroundColor="#EEE">
         <Section>
-          <Section.Row style={styles.centered}>
-            <Avatar size={80} source={avatar} onPress={() => screenProps.push('Profile')} />
-          </Section.Row>
-          <Section.Row style={styles.centered}>
-            <Section.Title>{fullName || ' '}</Section.Title>
-          </Section.Row>
-          <Section.Row style={styles.centered}>
-            <BigGoodDollar number={balance} />
-          </Section.Row>
-          <Section.Row style={styles.buttonRow}>
-            <PushButton routeName={'Send'} screenProps={screenProps} style={styles.leftButton}>
+          {scrollPos < 100 ? (
+            <>
+              <Section.Row style={styles.centered}>
+                <Avatar size={80} source={avatar} onPress={() => screenProps.push('Profile')} />
+              </Section.Row>
+              <Section.Row style={styles.centered}>
```

Comment:
this probably should be `<Section.Row justifyContent="center">

Reply:
Done